### ISSUE TITLE
Youtube | m.youtube.com - Support Youtube Mobile

### ISSUE THREAD

**Issue by xirpy** [PreMiD/Presences#8410]
### Presence name

Youtube

### Description

It's possible to use rich presence on mobile thanks to you guys.

The only problem is, the mobile version is not supported in the URL List, and only youtube.com works (using desktop layout)

It's possible for this to be added?

**Comment by armup31** [comment]
Yes, but ig the mobile site will have a different structure so we'll have to remake or make tweaks to the existing youtube presence

**Comment by ragxxv** [comment]
I second this as well, the original YouTube site kind of sucks.
